patchlevel.h

```diff
@@ -39,9 +39,32 @@ Instead use one of the version comparison macros.  See C<L</PERL_VERSION_EQ>>.
 /* do not adjust the whitespace! Configure expects the numbers to be
  * exactly on the third column */
 
-#define PERL_REVISION	5		/* age */
-#define PERL_VERSION	33		/* epoch */
-#define PERL_SUBVERSION	1		/* generation */
+/* Perl semantic versioning
+   you should avoid using these variables directly
+   and prefer using the compare functions.
+       - PERL_VERSION_EQ
+       - PERL_VERSION_NE
+       - PERL_VERSION_LT
+       - PERL_VERSION_GT
+       - PERL_VERSION_LE
+       - PERL_VERSION_GE
+*/
+#define PERL_VERSION_MAJOR	      7
+#define PERL_VERSION_MINOR	      1
+#define PERL_VERSION_PATCH	      0
+#define PERL_VERSION_RELEASE      0 /* for future usage */
+#define PERL_VERSION_BUILD        0 /* for future usage */
```

Comment:
Insert some more spaces to align these with the first three

Reply:
fixed by 09b8b94a01907c7ae7261328dbdd2a5db10fe5c9